pages/verification/index.tsx

```diff
@@ -3,16 +3,106 @@ import { Helmet } from "react-helmet";
 import React from "react";
 import LayoutContainer from "../../component/LayoutContainer";
 import Image from "next/image";
+import api from "../../shared/fetcher";
+import { useCheckExchangeTokenMutation } from "../../services/auth/authHooks";
+import { useRouter } from "next/router";
+import { GetServerSidePropsContext } from "next";
+import Cookies from "js-cookie";
+import { ITokenEnum } from "../../shared/enum";
+import { useInitChatAnonymousMutation } from "../../services/initChatAnonymous/initChatAnonymousHooks";
+import { toast } from 'react-toastify'
```

Comment:
please change relative import to alias import
e.g
import { BannerPage } from "component/Page/BannerPage"